samples/attested_tls/common/identity_verifier.cpp

```diff
@@ -19,74 +28,83 @@ oe_result_t enclave_claims_verifier_callback(
     oe_result_t result = OE_VERIFY_FAILED;
     const oe_claim_t* claim;
 
-    printf(TLS_CLIENT
+    printf(TLS_ENCLAVE
            "enclave_claims_verifier_callback is called with enclave "
            "identity information extracted from the evidence claims:\n");
 
     // Enclave's security version
     if ((claim = find_claim(
              claims, claims_length, OE_CLAIM_SECURITY_VERSION)) == nullptr)
     {
-        printf(TLS_CLIENT "could not find OE_CLAIM_SECURITY_VERSION\n");
+        printf(TLS_ENCLAVE "could not find OE_CLAIM_SECURITY_VERSION\n");
         goto exit;
     }
     if (claim->value_size != sizeof(uint32_t))
     {
         printf(
-            TLS_CLIENT "security_version size(%lu) checking failed\n",
+            TLS_ENCLAVE "security_version size(%lu) checking failed\n",
             claim->value_size);
         goto exit;
     }
-    printf(TLS_CLIENT "\nsecurity_version = %d\n", *claim->value);
+    printf(TLS_ENCLAVE "\nsecurity_version = %d\n", *claim->value);
 
     // The unique ID for the enclave, for SGX enclaves, this is the MRENCLAVE
     // value
     if ((claim = find_claim(claims, claims_length, OE_CLAIM_UNIQUE_ID)) ==
         nullptr)
     {
-        printf(TLS_CLIENT "could not find OE_CLAIM_UNIQUE_ID\n");
+        printf(TLS_ENCLAVE "could not find OE_CLAIM_UNIQUE_ID\n");
         goto exit;
     }
     if (claim->value_size != OE_UNIQUE_ID_SIZE)
     {
         printf(
-            TLS_CLIENT "unique_id size(%lu) checking failed\n",
+            TLS_ENCLAVE "unique_id size(%lu) checking failed\n",
             claim->value_size);
         goto exit;
     }
-    printf(TLS_CLIENT "\nverify unique_id:\n");
+#if defined(SERVER_CERT_VERIFY_CALLBACK)
```

Comment:
Again, consider actually encapsulating this as a function in a separate .cpp file, with appropriate version of the callback (seems like it's just a function rather than a callback) being compiled into the server/client builds.

Reply:
I will fix it as part of my future PRs related to openssl samples.